src/main/kotlin/uk/gov/justice/digital/assessments/services/RiskPredictorsService.kt

```diff
@@ -205,6 +205,10 @@ class RiskPredictorsService(
     return if (this == null) null else this == ResponseDto.YES.name
   }
 
+  fun String?.toPerpetratorBoolean(): Boolean? {
```

Comment:
this might need to change at some point when we do the changes for victim/perpetrator questions we have been discussing with Oasys guys but just a patch for now